playbooks/common/openshift-etcd/config.yml

```diff
@@ -0,0 +1,106 @@
+---
+- name: Gather and set facts for etcd hosts
+  hosts: oo_etcd_hosts_to_config
+  roles:
+  - openshift_facts
+  tasks:
+  - openshift_facts:
+      role: common
+      local_facts:
+        hostname: "{{ openshift_hostname | default(None) }}"
+  - name: Check for etcd certificates
+    stat:
+      path: "{{ item }}"
+    with_items:
+    - "/etc/etcd/ca.crt"
+    - "/etc/etcd/client.crt"
+    - "/etc/etcd/client.key"
+    - "/etc/etcd/peer-ca.crt"
+    - "/etc/etcd/peer.crt"
+    - "/etc/etcd/peer.key"
+    register: stat_result
```

Comment:
stat_result is effectively a global variable, so it should be called g_stat_result per the 
[openshift-ansible style guide](https://github.com/openshift/openshift-ansible/blob/master/docs/style_guide.adoc#ansible-global-variables)